### PR TITLE
Pages

### PR DESCRIPTION
**Pull request by DFurnes**
#### Changes
This pull request adds client-side routing, an example page, and a little navigation component for switching between them. You can check it out on the [review app](https://phoenix-next-pr-62.herokuapp.com/campaigns/teens-for-jeans).

#### How should this be reviewed?
I separated out the changes into a couple of smaller commits, so feel free to page through those one-by-one for more "bite sized" pieces. I'll make another PR for loading the additional page(s) & their content from Contentful.

### CODE REVIEW

**Comment by DFurnes** [comment]
Hmm, looks like the review app is having trouble fetching data from Contentful... looking into it. 🤔

**Comment by deadlybutter** [comment]
Was working a minute ago?

**Comment by deadlybutter** [comment]
Oh shit it might be, i was playing with the staging dyno...

**Comment by deadlybutter** [comment]
Nope, doesnt seem to be

**Comment by DFurnes** [comment]
Oh yep, broken on my local now too. Must've been cached.

**Comment by DFurnes** [comment]
Alright, rebased and fixed the missing content! I had borked the merge for #58 and was accidentally exporting the combined reducers rather than exporting each individually (and then combining them in the store).

**Comment by deadlybutter** [comment]
Treats for Troops still has a not found? https://phoenix-next-pr-62.herokuapp.com/campaigns/treats-troops

**Comment by deadlybutter** [comment]
It might just be missing something Contentful but wanted to double check

**Comment by DFurnes** [comment]
Yep! That's because I [hardcoded the base URL](https://github.com/DoSomething/phoenix-next/pull/62/files#diff-dbe454aeb5ceee5773a4e25dfe4e4d5cR17) for the router to Teens for Jeans for now. I'll fight out a better solution for that in a future PR but wanted to get this in so it's not blocking other work. 😄

File: resources/assets/containers/App.js [hunk 1]
```diff
@@ -1,19 +1,33 @@
...
+
+import Chrome from './Chrome';
 import Activity from './Activity';
+import ExamplePage from './ExamplePage';
+import NotFound from './NotFound';
 
-const store = configureStore({...reducers}, window.STATE);
+const store = configureStore({...reducers, routing: routerReducer}, window.STATE);
+const history = syncHistoryWithStore(useRouterHistory(createBrowserHistory)({
+  basename: '/campaigns/teens-for-jeans', // @TODO: Hardcoded.
```
Comment (deadlybutter) [inline resources/assets/containers/App.js]: Maybe replace this with `window.location.pathname` so it works across all campaigns?

Comment (DFurnes) [inline resources/assets/containers/App.js]: Yep, good call! Done in 5bddb89.

**Comment by deadlybutter** [comment]
Sweet, works across both campaigns now 🙌

**Pull request merged**